README.md

```diff
@@ -31,7 +31,54 @@ Trestle provides tooling to help orchestrate the compliance process across a num
 
 ## Important Note:
 
-The current version of trestle supports NIST OSCAL 1.1.2 as well as previous versions 1.1.x and 1.0.x. All files created by trestle will be output as OSCAL version 1.1.2.
+The current trestle v3:
+
+- supports NIST OSCAL 1.1.2 as well as previous versions
+- new OSCAL json documents created will be output as OSCAL version 1.1.2
+- old OSCAL json documents created using trestle v2 are supported (backwards compatibility)
+- some Python module dependencies have been upgraded
```

Comment:
I would recommend keeping the information about v3 in the README very high-level end user features. More granular changes like Python dep updates could be found in the release notes/changelog.

Reply:
So the above is OK, but...(next comment)

Reply:
Resolving this. See other comment.